src/sentry/static/sentry/app/views/settings/components/emptyMessage.jsx

```diff
@@ -1,12 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
 import styled from 'react-emotion';
 
-const EmptyMessage = styled.div`
-  display: flex;
-  justify-content: center;
```

Comment:
how does everything stay centered without this? the `icon` and `action` are block elements so they won't center with `text-align: center` correct?

Reply:
In a needlessly complex fashion. Fixed!